include/maliput/routing/derive_lane_s_routes.h

```diff
@@ -37,6 +37,12 @@
 namespace maliput {
 namespace routing {
 
+/// Returns the S coordinate in @p lane that is on the border with @p next_lane.
+/// When @p lane is not connected to @p next_lane , std::nullopt is returned.
+/// @throws maliput::common::assertion_error When @p lane is nullptr.
+/// @throws maliput::common::assertion_error When @p next_lane is nullptr.
+std::optional<double> DetermineEdgeS(const api::Lane* lane, const api::Lane* next_lane);
```

Comment:
Can both parameters be passed by const ref to avoid need to document and check for `nullptr`?

Reply:
[64bf98d](https://github.com/maliput/maliput/pull/619/commits/64bf98db8ea25dea6a6293ea3cdeac15172bf872)